Sources/TuistGraph/Models/RunActionOptions.swift

```diff
@@ -6,6 +6,9 @@ public struct RunActionOptions: Equatable, Codable {
     /// App Language.
     public let language: String?
 
+    /// App Region.
+    public let region: String?
```

Comment:
ouch this wasn't wired in all this time - good find 👍 

I think we're missing mapping this property in `RunActionOptions+ManifestMapper`

Reply:
Good catch, just please double check I am not sure it is all that should be done in https://github.com/tuist/tuist/pull/4983/commits/c88168ca1e56b1d01ed5e0724e4f76db6a6e452d